packages/suite/src/actions/wallet/blockchainActions.ts

```diff
@@ -138,7 +138,14 @@ export const subscribe = () => async (_dispatch: Dispatch, getState: GetState) =
         });
     });
 
-    return Promise.all(promises);
+    // does it belong here?
```

Comment:
yes, but consider to use already existing `Object.keys(sortedAccounts).map` above instead of creating another one